helm/druid/values.yaml

```diff
@@ -26,32 +26,37 @@ configMap:
   ##
   enabled: true
 
-## Define the key value pairs in the configmap
-configVars:
+## The configurations below would be set as environment variable across all pod
+globalConfig:
   ## DRUID env vars. ref: https://github.com/apache/druid/blob/master/distribution/docker/druid.sh#L29
-  # DRUID_LOG_LEVEL: "warn"
-  # DRUID_LOG4J: <?xml version="1.0" encoding="UTF-8" ?><Configuration status="WARN"><Appenders><Console name="Console" target="SYSTEM_OUT"><PatternLayout pattern="%d{ISO8601} %p [%t] %c - %m%n"/></Console></Appenders><Loggers><Root level="info"><AppenderRef ref="Console"/></Root><Logger name="org.apache.druid.jetty.RequestLog" additivity="false" level="DEBUG"><AppenderRef ref="Console"/></Logger></Loggers></Configuration>
-  DRUID_USE_CONTAINER_IP: "true"
+  DRUID_USE_CONTAINER_IP: 'true'
 
   ## Druid Common Configurations. ref: https://druid.apache.org/docs/latest/configuration/index.html#common-configurations
-  druid_extensions_loadList: '["druid-histogram", "druid-datasketches", "druid-lookups-cached-global", "postgresql-metadata-storage"]'
-  druid_metadata_storage_type: postgresql
-  druid_metadata_storage_connector_connectURI: jdbc:postgresql://postgres:5432/druid
-  druid_metadata_storage_connector_user: druid
-  druid_metadata_storage_connector_password: druid
+  druid_extensions_loadList: '["druid-datasketches", "druid-lookups-cached-global", "postgresql-metadata-storage"]'
```

Comment:
what is the reason for removing `druid-histogram` from here?

Reply:
Because the [doc](http://druid.apache.org/docs/latest/development/extensions-core/approximate-histograms.html) says that it's deprecated and should use druid-datasketches instead.